### PR TITLE
New serverless pattern - CloudWatch Logs subscription for Lambda - CDK

### PR DESCRIPTION
**Pull request by pubudusj**
*Issue #, if available:*
N/A

*Description of changes:*
Example added for CloudWatch Logs subscription for Lambda with CDK.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cloudwatch-logs-subscription-lambda-cdk/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Amazon CloudWatch Logs Subscription to AWS Lambda with CDK
+
+This pattern demonstrates how to create the Amazon CloudWatch Log Subscription to AWS Lambada. Logs filtering is also demonstrated to filter only a certain logs sent to AWS Lambda.
```
Comment (mavi888) [inline cloudwatch-logs-subscription-lambda-cdk/README.md:3]: typo - Lambada

Comment (mavi888) [inline cloudwatch-logs-subscription-lambda-cdk/README.md:3]: rewrite - only a certain logs

Comment (pubudusj) [reply cloudwatch-logs-subscription-lambda-cdk/README.md]: This is updated.

**Comment by mavi888** [comment]
Thanks for your submission, you can find the published pattern here: https://serverlessland.com/patterns/cloudwatch-logs-subscription-lambda-cdk

**Pull request merged**